### DISCUSSION TITLE
Indication diodes for outputs

### DISCUSSION THREAD

**Discussion by mDubovsky** [pilotak/eth-io#9] [Ideas]
Not the easiest task for sure, but I'd take this one if approved.
While testing, whether it be testing of the PCB (before putting it into the case) or testing of the whole system, you have to measure continuity before ground and outputs to tell if they're switched.
It would be beneficial to have an (optional) output indication diodes. This can be done with resistor nets and light guides to make the assembly easier. As with the #6 people who don't want this can just omit the assembly of said components.

It can be beneficial when the device is commissioned as well for people who don't have relays with indication diodes.
I can imagine the LED light guides to end here, on the edge of the second "level" of the case (just imagine this is the output side :) ):
![image](https://user-images.githubusercontent.com/9304179/208431707-5ee1607c-c2bc-4737-8059-3c0aaf0f2c8c.png)

**Comment by pilotak** [comment]
The biggest disadvantage is that you have to connect the the power supply to the LEDs in order to light them - if the output is off this voltage will flow through the diode right to the output pin (with limited current). So if you have a mixed power supplies ie some relays/components that are 9V input, some 12V and some 24V your are in trouble. The indication would need to be at the IN of the mosfet which doesn't nessesary mean the output is really ON when the mosfet is damaged. 

The indication would be also perfect on the inputs side.

Perfect idea, but impossible due to no space left from the top side

**Reply by mDubovsky** [reply]
Hmm, you're right. I didn't think about that. Though I would love to have the indication for debug purposes it seems that for now it's not really feasible.
